src/styles/components/_list-button.scss

```diff
@@ -0,0 +1,54 @@
+.btn-list-button {
+  @include state-styling($wit, $grijs-20, $grijs-90);
+  font-weight: 600;
+  margin-top: ($u1 / 2);
+  padding: #{$u1 + 2} $u2;
+  text-align: left;
+  white-space: normal;
+  width: 100%;
+
+  &:first-of-type {
+    margin-top: 0;
+  }
+
+  &:hover,
+  &:focus {
+    @include state-styling($wit, $dso-button-default-hover-color, $grijs-90);
+  }
+
+  &:active,
+  &.active {
+    &,
+    &:hover,
+    &:not(.btn):visited {
```

Comment:
Hoezo `:not(.btn)` ?

Reply:
Sterker nog: hoezo `:visited` op een `button`? Weggehaald...